### PR TITLE
open file in export

### PR DESCRIPTION
**Pull request by tuannv007**

### CODE REVIEW

File: app/src/main/java/com/framgia/fdms/screen/profile/export/ExportContract.java [hunk 1]
```diff
@@ -12,12 +12,16 @@
         void exportPdf();
 
         void onDestroy();
+
+        void OpenFilePDF();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/profile/export/ExportContract.java]: convention

Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/profile/export/ExportContract.java]: onpenFile(string filePath)

File: app/src/main/java/com/framgia/fdms/screen/profile/export/ExportViewModel.java [hunk 2]
```diff
@@ -44,4 +49,14 @@ public void exportPdf() {
     public void onDestroy() {
         mPresenter.onDestroy();
     }
+
+    @Override
+    public void openFilePDF(String filePath) {
+        File file = new File(filePath);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/profile/export/ExportViewModel.java]: pls check file exit

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -186,4 +186,5 @@
     <string name="msg_no_request_manager">Not request device</string>
     <string name="msg_request_write_file">Write list device using</string>
     <string name="title_used">Used</string>
+    <string name="title_open_with">OPen with</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml]: open > Open

File: app/src/main/java/com/framgia/fdms/screen/profile/export/ExportViewModel.java (hunk 2 above)
Comment (tuannv007) [inline app/src/main/java/com/framgia/fdms/screen/profile/export/ExportViewModel.java]: done

**Pull request merged**